added meta html values (#3213)
plone/volto: docs/source/recipes/appextras.md

---
html_meta:

>   "description": "The `AppExtras` component is a general use insertion point for things like

Reviewer: ```suggestion
  "description": "The `AppExtras` component is a general use insertion point for general purpose code spanning the whole application or for third party services code."
```

> analytics, general purpose code spanning the whole application or third party services code."

Reviewer: ```suggestion
```

>   "property=og:description": "The `AppExtras` component is a general use insertion point for things like analytics, general purpose code spanning the whole application or third party services code."

Reviewer: ```suggestion
  "property=og:description": "The `AppExtras` component is a general use insertion point for general purpose code spanning the whole application or for third party services code."
```
